fickling/exception.py

```diff
@@ -6,3 +6,12 @@ def __init__(self, filepath, info):
 
     def __str__(self):
         return f"Safety results for {self.filepath} : {str(self.info)}"
+
+
+class WrongMethodError(Exception):
+    def __init__(self):
+        super().__init__()
+
+    def __str__(self):
+        return """This method has been removed. Use fickling.is_likely_safe()
+        on the pickle file or analysis.check_safety() on the Pickled object"""
```

Comment:
Nice to have a dedicated exception. However, the exception message should be an argument of the exception, not hardcoded in the class.
```suggestion
class WrongMethodError(Exception):
    def __init__(self, msg):
        super().__init__()
        self.msg = msg

    def __str__(self):
        return self.msg
```